book_homework5/book_homework5/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿book_homework5.Book book1 = new book_homework5.Book ("Neznajka",167,"123ADF125av4") ;
+book_homework5.Book book2 = new book_homework5.Book ( "Vij",58,"dvdffv12ds" );
+book_homework5.Book book3 = new book_homework5.Book ("Tom Sojer", 132, "987654dvd" );
+book_homework5.Book book4 = new book_homework5.Book ("Dinka", 334,"85jfuie45" );
+book_homework5.Book book5 = new book_homework5.Book ( "ABC",88, "09dneuie34");
+
+book_homework5.Book[] books = { book1, book2, book3, book4, book5 };
+foreach (book_homework5.Book item in books)
+{
+    item.displayBooks();
+}
```

Comment:
add new line at the end. code style

Reply:
1) .gitignore was added to this PR, moreover it contains files which should not be in the request but they are. Please, could you clear what is the issue with my .gitignore file. 
2) Please, could you  clear where the new lines should be added or what is the particular naming issues or give the link to read. The code style is out of our lessons' topics.